main.go

```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
+	"github.com/divoxx/terraform-provider-helm/helm"
```

Comment:
I needed to change it so that the correct package gets compiled when testing, so it's correct in the context of the `divoxx/terraform-provider-helm` context (we are using that branch).

My thought was to change it back when/if this is ready to be merged. Does that work?

Reply:
Yeah, this would work in this case since this project is non go-gettable. It's just force of habit when working on golang forks, to make sure the repo is still go-gettable and correctly loads the packages from the repo instead of pulling from the original.

I'll update.